logging/bin/deploy_esexporter.sh

```diff
@@ -107,6 +135,7 @@ helm $helmDebug upgrade --install es-exporter \
  -f $primaryValuesFile \
  -f $wnpValuesFile \
  -f $openshiftValuesFile \
+ -f $airgapValuesFile \
```

Comment:
Don't we need/want to reference $AIRGAP_HELM_REPO below (on line 140)?   Hmmm, not sure why the \-\-set fullnameOverride option appears _after_ the name of the chart.  That's inconsistent with the ordering of args in all of our other Helm chart deployment calls.  Probably should move it to be consistent with the others.

Reply:
We do need to add the $AIRGAP_HELM_REPO ... I might've missed this one because it was kind of flushed in like that.  I moved it to its own line and added the $AIRGAP_HELM_REPO parameter to it.  It will show up in the next push.